### PR TITLE
Add recipe funcannot v2.8

### PR DESCRIPTION
**Pull request by mtekman**
* [X ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/funcannot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  md5: 2e70b503a1632998deba759b50c0c07f
+   
+
+build:
+  number: 0
+  #skip: True # [osx]
+
+requirements:
+  build:
+    - gcc
```
Comment (bgruening) [inline recipes/funcannot/meta.yaml]: llvm on macos is not working?

File: recipes/funcannot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - zlib {{CONDA_ZLIB}}*
```
Comment (bgruening) [inline recipes/funcannot/meta.yaml]: is this really needed or linked in statically? I'm just wondering why it is not needed as runtime dependency.

File: recipes/funcannot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    - qt
+
+  run:
+    - libgcc
+    - qt
```
Comment (bgruening) [inline recipes/funcannot/meta.yaml]: do you need to pin the qt version? version 4 or 5?

File: recipes/funcannot/meta.yaml (hunk 1 above)
Comment (mtekman) [reply recipes/funcannot/meta.yaml]: untested, but if it can build under gcc, why not use gcc?

File: recipes/funcannot/meta.yaml (hunk 3 above)
Comment (mtekman) [reply recipes/funcannot/meta.yaml]: It can build for both, but I'll lock it qt4 for now

File: recipes/funcannot/meta.yaml (hunk 2 above)
Comment (mtekman) [reply recipes/funcannot/meta.yaml]: Ah. I thought zlib was needed by conda at build time to unpack the tool. 
So these are includes that are fed to the actual build script, okay I understand -- will remove.

File: recipes/funcannot/meta.yaml (hunk 1 above)
Comment (mtekman) [reply recipes/funcannot/meta.yaml]: (I will specify llvm for osx, in future commits - I just want to know why llvm is preferred over a more cross-platform soln)

**Pull request merged**